### PR TITLE
doc: add Intro to README

### PR DESCRIPTION
**Pull request by LeonidVas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +29,60 @@ Tarantool-based applications.
...
 * [Commands](#commands)
 
+## Intro
+
+`tt` is tarantool's instance and environment management utility and is used to develop, deploy, run and operate applications.
+
+One of the basic concepts that `tt` introduces is "environment". The "environment" is
+an isolated workspace for the tarantool application suite.
+[`tt.yaml` configuration file](#configuration) is define the root and configuration of
+the *environment*. When `tt` installed from a repository by a package manager (`apt`,
```
Comment (psergee) [inline README.md:39]:
```suggestion
the *environment*. When `tt` is installed from a repository by a package manager (`apt`,
```

File: README.md [hunk 2]
```diff
@@ -28,6 +29,60 @@ Tarantool-based applications.
...
+`rpm`, ...) a "system" config file (`/etc/tarantool/tt.yaml`) is included which forms
+the "system" environment - the case when `tt` replace the
```
Comment (psergee) [inline README.md:41]:
```suggestion
the "system" environment - the case when `tt` replaces the
```

File: README.md [hunk 3]
```diff
@@ -28,6 +29,60 @@ Tarantool-based applications.
   * [Setting Tarantool configuration parameters via environment variables](#setting-tarantool-configuration-parameters-via-environment-variables)
 * [Commands](#commands)
 
+## Intro
+
+`tt` is tarantool's instance and environment management utility and is used to develop, deploy, run and operate applications.
+
+One of the basic concepts that `tt` introduces is "environment". The "environment" is
+an isolated workspace for the tarantool application suite.
+[`tt.yaml` configuration file](#configuration) is define the root and configuration of
```
[the remark this answers is not in the document]
Comment (LeonidVas) [reply README.md]: Updated.

File: README.md (hunk 2 above)
Comment (LeonidVas) [reply README.md]: Updated.

**Review by psergee — APPROVED** [review]
thanks for the patch, lgtm

**Pull request merged**